guide/pt/ch_oview.xml

```diff
@@ -39,17 +39,17 @@
     <screenshot id="oview-intro">
       <mediaobject>
         <imageobject role="html">
-          <imagedata fileref="figures/oview_intro.png" format="PNG"
-                     srccredit="Pedro Albuquerque" width="510"></imagedata>
+          <imagedata fileref="figures/oview_intro.png" 
+						format="PNG" srccredit="Pedro Albuquerque" width="510px"/>
```

Comment:
1.  `format="PNG"` is not required. It follows from the filename extension.
2.  In between we use ` width="&img-w;"` to have a unique max width and can change it at one place. And it is only required for image width > 800.